repo/blob/b2/b2_storage.go

```diff
@@ -261,7 +261,7 @@ func toBandwidth(bytesPerSecond int) iothrottler.Bandwidth {
 	return iothrottler.Bandwidth(bytesPerSecond) * iothrottler.BytesPerSecond
 }
 
-// New creates new B2-backed storage with specified options:
+// New creates new B2-backed storage with specified options:.
```

Comment:
May not be formatted correctly
```suggestion
// New creates new B2-backed storage with specified options.
```